etl/src/datatreatments/PopulateGeoCentroid.ts

```diff
@@ -0,0 +1,80 @@
+import { AbstractDatatreatment } from '../common/AbstractDatatreatment';
+
+export class PopulateGeoCentroid extends AbstractDatatreatment {
```

Comment:
Du coup ce script est joué à chaque migration ? Est-ce que ça ne créé pas des doublons des centroides ?

Reply:
Oui, il est joué à chaque migration. Certes, ça rallonge le temps d'exécution de 2mn environ mais ça à l'avantage de ne pas rajouter une migration chaque année. Et il n'y à pas de doublons inséré dans la table grâce à la vérification de la clé d'unicité quand il y a conflit:
ON CONFLICT 
 ON CONSTRAINT ${this.table}_unique_key
 DO NOTHING;